modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -988,6 +988,55 @@ definition {
 		}
 	}
 
+	@description = "LPS-156566 - Verify an Action with Groovy Script can be triggered after deleting an entry."
+	@priority = "4"
+	test CanTriggerActionDeleteWithGroovyScript {
+		task ("Given an object witha a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 156",
+				objectName = "CustomObject156",
+				pluralLabelName = "Custom Objects 156");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Integer",
+				fieldName = "customInteger",
+				fieldType = "Integer",
+				isRequired = "false",
+				objectName = "CustomObject156");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject156");
+		}
+
+		task ("When an action is created using Groovy Script and is triggered when an object field is deleted") {
```

Comment:
the same thing here, it can be something like: "When an action is created using Groovy Script and is triggered when an object entry is deleted"